### PR TITLE
Hero

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

**Comment by GresosJ** [comment]
Still it isnt reponsive because we dont have a design for mobile

File: app/layout.tsx [hunk 1]
```diff
@@ -16,10 +15,7 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>
-        <Navbar />
-        {children}
-      </body>
```
Comment (EnzoVieira) [inline app/layout.tsx:22]: Why did you removed the Navbar from the Root Layout?

Comment (GresosJ) [reply app/layout.tsx:22]: I thought it would be better on the page, as there may be pages that don't use the Navbar

Comment (RuiL1904) [reply app/layout.tsx:22]: Could you suggest which pages would not make use of the navbar?

Comment (GresosJ) [reply app/layout.tsx:22]: Like Error page

Comment (GresosJ) [reply app/layout.tsx:22]: But with the next 13 dont make to much sense what I said

File: components/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          unoptimized
+        />
+        <Image
+          className="absolute left-1/2 top-1/2 h-full w-full -translate-x-1/2 -translate-y-1/2 scale-150 transform rounded-lg object-cover opacity-40 blur-3xl filter"
+          src={logo}
+          alt=""
+          width={300}
+          height={343}
+          priority
+          unoptimized
```
Comment (MarioRodrigues10) [inline components/Hero.tsx:29]:
```suggestion
```

File: components/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        </div>
+      </div>
+      <div className="relative">
+        <Image
+          src={logo}
+          alt=""
+          width={300}
+          height={343}
+          priority
+          unoptimized
```
Comment (MarioRodrigues10) [inline components/Hero.tsx:20]:
```suggestion
          unoptimized
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Please look into this minor changes :pray:

**Review by MarioRodrigues10 — APPROVED** [review]
Good job :rocket:

**Pull request merged**